docs/source/server/concurrency.rst

```diff
@@ -0,0 +1,43 @@
+Multi-Tenancy & Concurrency
+==========================
+
+Graphistry `register()` and `plot()` are stateful. Methods are the concurreny boundary.
+
+This is because on `plot`, the `ArrowUploader` manages user auth token refresh on session
+state. 
+
+`Plottable._pygraphistry.session` = `Graphistry.session`
```

Comment:
Discussion of internal details seems confusing for user docs

Reply:
moved it